<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Network_Key</class>
 <widget class="QMainWindow" name="Network_Key">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Escalation Ticket - Key Upgrade</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QGroupBox" name="Ticket_Buttons">
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QPushButton" name="Copy">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Copy to Clipboard</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="Clear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Generate Ticket</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>510</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>368</width>
         <height>508</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="ticketInfo">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;Ticket Information:           &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_cstname">
          <property name="text">
           <string>Customer Name:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLineEdit" name="cstName"/>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_cstid">
          <property name="text">
           <string>Customer ID:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLineEdit" name="cstID"/>
        </item>
        <item row="13" column="0">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_purpose">
          <property name="text">
           <string>Aggregate of Radio:</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLineEdit" name="tktAggregate">
          <property name="placeholderText">
           <string>Current key on radio.</string>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_managed">
          <property name="text">
           <string>Manage IP:</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLineEdit" name="tktRadioIP"/>
        </item>
        <item row="24" column="0">
         <widget class="QLabel" name="label_routermac">
          <property name="text">
           <string>Radio Mac Address:</string>
          </property>
         </widget>
        </item>
        <item row="25" column="0">
         <widget class="QLineEdit" name="tktRouterMAC">
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="26" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="33" column="0">
         <widget class="QLabel" name="label_notes">
          <property name="text">
           <string>Notes:</string>
          </property>
         </widget>
        </item>
        <item row="34" column="0">
         <widget class="QTextEdit" name="tktNotes">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Enter any helpful information that may be important, here.</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>ticketInfo</zorder>
       <zorder>label_notes</zorder>
       <zorder>tktNotes</zorder>
       <zorder>label_cstname</zorder>
       <zorder>cstName</zorder>
       <zorder>verticalSpacer_6</zorder>
       <zorder>tktAggregate</zorder>
       <zorder>label_purpose</zorder>
       <zorder>label_managed</zorder>
       <zorder>tktRadioIP</zorder>
       <zorder>tktRouterMAC</zorder>
       <zorder>label_routermac</zorder>
       <zorder>verticalSpacer_3</zorder>
       <zorder>cstID</zorder>
       <zorder>label_cstid</zorder>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTextEdit" name="ticketTemplate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>cstName</tabstop>
  <tabstop>cstID</tabstop>
  <tabstop>tktAggregate</tabstop>
  <tabstop>tktRadioIP</tabstop>
  <tabstop>tktRouterMAC</tabstop>
  <tabstop>tktNotes</tabstop>
  <tabstop>ticketTemplate</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>Copy</tabstop>
  <tabstop>Clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
